DataDefinitions/SystemState.cs

```diff
@@ -29,10 +30,6 @@ static SystemState()
             var Investment = new SystemState("Investment");
         }
 
-        public static readonly SystemState None;
```

Comment:
Sorry, not keen on this. Referencing a variable is way faster than doing a linear lookup on EDName.